setup/src/Magento/Setup/Model/ConfigOptionsList/Directories.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Setup\Model\ConfigOptionsList;
+
+use Magento\Framework\App\DeploymentConfig;
+use Magento\Framework\Config\ConfigOptionsListConstants;
+use Magento\Framework\Config\Data\ConfigData;
+use Magento\Framework\Config\File\ConfigFilePool;
+use Magento\Framework\Exception\InputException;
+use Magento\Framework\Setup\ConfigOptionsListInterface;
+use Magento\Framework\Setup\Option\SelectConfigOption;
+
+/**
+ * Deployment configuration options for the document root root
+ */
+class Directories implements ConfigOptionsListInterface
+{
+    public const INPUT_KEY_DOCUMENT_ROOT_IS_PUB = 'document-root-is-pub';
+
+    /**
+     * @inheritdoc
+     */
+    public function getOptions()
```

Comment:
Can we define here return type?
```suggestion
    public function getOptions(): array
```